src/assets/sass/protocol/components/_breadcrumb.scss

```diff
@@ -0,0 +1,36 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../includes/lib';
+
+.mzp-c-breadcrumb {
+    background-color: #f3f3f3;
+    margin-bottom: 0;
+    padding: $spacing-sm $spacing-md;
+
+    .mzp-c-breadcrumb-item {
+        font-weight: bold;
```

Comment:
I think we're overriding the bold font type in the `-item`, `-title` selector below with 400 font weight

Reply:
I wonder too if it's worthwhile having separate classes for item and title or if we could use `:first-child` and `.mzp-c-breadcrumb-item + .mzp-c-breadcrumb-item` selectors to target the first and subsequent items in a way that could scale to multiple items in the breadcrumb.